src/frontend/src/lib/components/transactions/AllTransactions.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import TransactionsPlaceholder from '$lib/components/transactions/TransactionsPlaceholder.svelte';
+	import { SLIDE_DURATION } from '$lib/constants/transition.constants';
+	import { enabledTokens } from '$lib/derived/tokens.derived';
+	import type { UnifiedTransactionUi } from '$lib/types/transaction';
+	import {
+		isNetworkIdBTCMainnet,
+		isNetworkIdEthereum,
+		isNetworkIdICP
+	} from '$lib/utils/network.utils';
+
+	let transactions: UnifiedTransactionUi[];
+	$: transactions = $enabledTokens.reduce<UnifiedTransactionUi[]>(
+		(acc, { network: { id: networkId } }) => {
+			if (isNetworkIdBTCMainnet(networkId)) {
+				// TODO: Implement BTC transactions
+			}
+
+			if (isNetworkIdICP(networkId)) {
+				// TODO: Implement ICP transactions
+				return acc;
+			}
+
+			if (isNetworkIdEthereum(networkId)) {
+				// TODO: Implement Ethereum transactions
+				return acc;
+			}
+
+			return acc;
+		},
+		[]
+	);
+</script>
+
+{#if transactions.length > 0}
+	{#each transactions as transaction, index (`${transaction.id}-${index}`)}
+		<li in:slide={SLIDE_DURATION}>
+			<svelte:component this={transaction.transactionComponent} {transaction} />
+		</li>
+	{/each}
+{/if}
+
+{#if transactions.length === 0}
```

Comment:
A test for this? E2E or vite

"it should display the placeholder if there is no transactions"

Reply:
the E2E test already exists, in the sense, it will always show empty list, for now

but I created the vite test too